docs/migration-guide/to-16.md

```diff
@@ -154,6 +154,16 @@ We've published a new version of [Jest preset](https://github.com/stylelint/jest
 }
 ```
 
+If you get a segmentation fault while running the preset on Node.js 18, you can use [jest-light-runner](https://www.npmjs.com/package/jest-light-runner):
+
+```diff json
+export default {
```

Comment:
[ask] Is `diff json` incorrect? The snippet seems JS code.

Reply:
Oops, good catch.